cEpiabm/examples/basic_example/main.cpp

```diff
@@ -69,17 +69,17 @@ void run()
 
 int main()
 {
-    try
-    {
+    //try
+    //{
         run();
```

Comment:
Could we keep just this line and remove all the comments?

Reply:
I think it was left as an example, but happy to remove now we have logging for cEpiabm too